src/patterns/fundamentals/grid/stack-object.hbs

```diff
@@ -0,0 +1,17 @@
+---
+name: Stack Object
+description: The Stack object lays items out in a column.
```

Comment:
I think we need to expand this a little. Something about how the Stack Object is not a grid in itself, but a foundation that can be used to create a grid. I'm not exactly sure on wording, but calling this page "Grid" and then not really getting into what people expect from a grid until half way down the page might be confusing.

Reply:
Great point, I'll elaborate.